My-Spec-Tutorial.package/RadioButtonGroup.extension/instance/selectedButton.st

```diff
@@ -0,0 +1,4 @@
+*My-Spec-Tutorial
+selectedButton
+
+	^ buttons detect: [ :b | b state ] ifNone: [ nil ]
```

Comment:
Yes that problem is resolved. Can you tell me more about the last part.I think you are trying to say that there should a module of "detect:" for "RadioButtionGroup". So that we don't have to write switch statements. Or any thing else you want to say. 
Please tell me how can i contact with you? I have some questions.